pkg/cluster/docker_desktop_dial.go

```diff
@@ -18,17 +18,26 @@ func dockerDesktopSocketPaths() ([]string, error) {
 		return nil, err
 	}
 
-	return []string{
-		// Older versions of docker desktop use this socket.
-		filepath.Join(homedir, "Library/Containers/com.docker.docker/Data/gui-api.sock"),
-
-		// Newer versions of docker desktop use this socket.
-		filepath.Join(homedir, "Library/Containers/com.docker.docker/Data/backend.native.sock"),
-	}, nil
+	switch runtime.GOOS {
+	case "darwin":
+		return []string{
+			// Older versions of docker desktop use this socket.
+			filepath.Join(homedir, "Library/Containers/com.docker.docker/Data/gui-api.sock"),
+
+			// Newer versions of docker desktop use this socket.
+			filepath.Join(homedir, "Library/Containers/com.docker.docker/Data/backend.native.sock"),
+		}, nil
+	case "linux":
+		return []string{
+			// Docker Desktop for Linux
+			filepath.Join(homedir, ".docker/desktop/backend.native.sock"),
```

Comment:
I was able to verify this location works in a VM, I just can't get DD4L fully started in a VM yet

Reply:
Also, I'm going to just switch these to `backend.sock` since the native socket is going away.